app/src/main/java/net/gsantner/opoc/util/GsContextUtils.java

```diff
@@ -2878,10 +2878,10 @@ public boolean isDarkModeEnabled(final Context context) {
         return false;
     }
 
-    public static void blinkView(final View view) {
+    public static void flashView(final View view) {
         if (view != null) {
-            final float init = view.getAlpha();
-            ObjectAnimator.ofFloat(view, View.ALPHA, init, 0.1f, 1.0f, 0.1f, 1.0f, init).setDuration(1000).start();
+            view.setBackgroundColor(Color.LTGRAY); // Highlight
+            view.postDelayed(() -> view.setBackgroundColor(Color.TRANSPARENT), 300);
```

Comment:
suggestion: wrap the second setBackground call into a try-catch. So just in case the view already went away, it doesn't crash here.

Reply:
I have reverted it to the old blink animation as it was way less jarring

![image](https://github.com/gsantner/markor/assets/1940940/8016f5d4-3a96-401b-9b00-15185893a8c7)

Reply:
checked it, it's noticeable but not disturbing